lib/galaxy/workflow/refactor/schema.py

```diff
@@ -217,7 +217,7 @@ class UpgradeToolAction(BaseAction):
     ConnectAction,
     DisconnectAction,
     ExtractInputAction,
-    ExtractLegacyParameter,
+    ExtractImplicitParameter,
```

Comment:
This needs to target 21.01 if you're changing the API.

Reply:
Yes, it should target 21.01. I wanted to replace the generic term `legacy` here, since it does not really convey any information and we are somewhat over using it. Thanks a lot for looking into it. I am definitely open to suggestions.

Reply:
Implicit sounds nifty and exciting and new - and what it is is parameters that don't work on the backend or with the API. I too wish we would have argued about this sooner, I understand I did slip this in over the holidays which was shitty of me.

Reply:
Renamed it `Untyped` internally following @mvdbeeks suggestion. Readded `legacy` to the UI help text.